src/GatewayAPI/Controllers/TransactionController.cs

```diff
@@ -167,25 +164,20 @@ public async Task<TransactionStatusResponse> GetTransactionStatus(TransactionSta
     }
 
     [HttpPost("build")]
-    public async Task<TransactionBuildResponse> BuildTransaction(TransactionBuildRequest request)
+    public Task<TransactionBuildResponse> BuildTransaction(TransactionBuildRequest request)
     {
-        var ledgerState = await _ledgerStateQuerier.GetValidLedgerStateForConstructionRequest(request.NetworkIdentifier, request.AtStateIdentifier);
-        return new TransactionBuildResponse(
-            await _constructionAndSubmissionService.HandleBuildRequest(request, ledgerState)
-        );
+        throw InvalidRequestException.FromOtherError("Olympia Network has been decommissioned.", "TX build attempt");
```

Comment:
I believe this is a user facing message? It may even be passed through to users in the Olympia wallet.

In which case, I'd rather it were a little less scary, eg:

`"Radix has now migrated to Babylon, but you are attempting to create a transaction against the old Olympia version, which isn't possible. Please see https://wallet.radixdlt.com/ to get an updated wallet or https://docs.radixdlt.com/ for documentation concerning integrating with Babylon."`

Reply:
Updated.